python/taichi/_kernels.py

```diff
@@ -179,6 +179,29 @@ def ext_arr_to_matrix(arr: ndarray_type.ndarray(), mat: template(),
                     mat[I][p, q] = arr[I, p, q]
 
 
+@kernel
+def ext_arr_from_vulkan_layout_to_normal_layout(
+        vk_image: ndarray_type.ndarray(),
+        normal_image: ndarray_type.ndarray()):
+    '''extract ndarray of raw vulkan memory layout to normal memory
+    layout. Explanation of why using ti.ndarray:
+        ti.field struct has a complicated memory layout under unpacked
```

Comment:
OOC: why do we want to mention `ti.field` in this docstring? I'd expect this one to explain what's vulkan memory layout   and normal layout means :P